akka-docs/src/test/scala/docs/stream/operators/source/Zip.scala

```diff
@@ -60,4 +60,17 @@ object Zip {
     // (4,default)
     // #zipAll-simple
   }
+
+  def zipLatest(): Unit = {
+    // #zipLatest-example
+    val numbers = Source(1 :: 2 :: Nil)
+    val letters = Source("a" :: "b" :: Nil)
+
+    numbers.zipLatest(letters).runForeach(println)
+    // this will print
+    // Pair(1,a)
+    // Pair(2,a)
+    // Pair(2,b)
+    // #zipLatest-example
```

Comment:
No `Pair` in Scala, instead it will be a tuple where `toString` will be like this: `(1,a)`

It might end up with the same output each time since fused into a single stream here, but It's not generally deterministic though, so maybe worth mentioning it could also print this if `b` arrives before `2`:
```
(1, a)
(1, b)
(2, b)
```

Reply:
makes sense! it's non-determinstic based on what element from the streams are available.

Reply:
@johanandren : do you know if there's a way to run Scala tests in sbt but not the multi-jvm ones but others?

I couldn't tests Scala code because I didn't found a command to run a single test compared to all tests in a file

Reply:
You can run a specific test case with `akka-stream-tests/testOnly a.b.c.TestClass` and a specific test int that class with `akka-stream-tests/testOnly a.b.c.TestClass -- -z "some text from test name"`